transport/src/main/java/io/netty/channel/DefaultChannelPipeline.java

```diff
@@ -244,7 +396,7 @@ public ChannelPipeline addFirst(EventExecutorGroup group, ChannelHandler... hand
 
         for (int i = size - 1; i >= 0; i --) {
             ChannelHandler h = handlers[i];
-            addFirst(group, generateName(h), h);
```

Comment:
hard to see with the github diff but is the generateName handled somewhere else now when passed as null, or?